app/(pages)/content/[collection]/edit/_components/DynamicForm/DynamicForm.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import dynamic from 'next/dynamic';
+import styles from './DescriptionForm.module.scss';
+import {
+  FieldType,
+  FormSection,
+  Field,
+  FileItem,
+} from '../../../../../../../type';
```

Comment:
You can make a shortcut for this. Kinda like how we have it for `@components or @globals`